### ISSUE TITLE
Tests are failing with "cannot import name 'RawContent'" error

### ISSUE THREAD

**Issue by apryiomka** [meta-llama/llama-stack#647]
### System Info

Wondows/WSL Unubntu 22

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### 🐛 Describe the bug

The `RawContent` is not in the latest PyPi release

### Error logs

ImportError: Error importing plugin "llama_stack.providers.tests.inference.fixtures": cannot import name 'RawContent' from 'llama_models.llama3.api.datatypes'

### Expected behavior

the tests should reference the released version of the module

**Comment by ashwinb** [comment]
@apryiomka yeah we are about to release the new packages. when we make big changes which depend on llama-models updates, there's a small window of such issues. Packages are imminent (within next couple of hours)

**Comment by hardikjshah** [comment]
@apryiomka do you still see this issue ?

**Comment by hardikjshah** [comment]
Going to close issue since seems like an old issue, please reopen a new one if you continue to see this.
